ramls/examples/name.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
The structure of the tags must be the same as the bibliographic ones.
Some properties are missing for both fixed and variable tags, 
see record.json of bibliographic records in tests, example:
 {
        "code": "008",
        "mandatory": true,
        "fieldStatus": "unchanged",
        "fixedField": {
          "categoryCode": 1,
          "headerTypeCode": 31,
          "code": "008",
          "displayValue": "180924s        enk           000 u eng r",
          "sequenceNumber": 0,
          "attributes": {}
        },



 {
        "code": "100",
        "mandatory": false,
        "fieldStatus": "new",
        "variableField": {
          "keyNumber": 0,
          "code": "100",
          "ind1": "1",
          "ind2": " ",
          "displayValue": "\u001faAlessandro Manzoni,\u001fd1785-1873.",
          "headingTypeCode": 0,
          "sequenceNumber": 0,
          "skipInFiling": 0,
          "subfields": [],
          "categoryCode": 2
        },
        "added": true
      },

Reply:
I have done the changes